File: app/src/main/java/com/nononsenseapps/feeder/ui/DebugLogActivity.java
package com.nononsenseapps.feeder.ui;

import android.os.Bundle;
import android.widget.TextView;
import com.nononsenseapps.feeder.R;
import com.nononsenseapps.feeder.util.FileLog;


public class DebugLogActivity extends BaseActivity {

    @Override
    public void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.debug_log);
    }

    @Override
    public void onResume() {
        FileLog log = FileLog.singleton.getInstance(this);

        ((TextView) findViewById(android.R.id.text1)).setText(log.getLog());

        super.onResume();
    }
}

File: app/src/main/java/com/nononsenseapps/feeder/ui/DebugLogActivity.kt
package com.nononsenseapps.feeder.ui

import android.os.Bundle
import android.widget.TextView
import com.nononsenseapps.feeder.R
import com.nononsenseapps.feeder.util.FileLog


class DebugLogActivity : BaseActivity() {

    public override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.debug_log)
    }

    override fun onResume() {
        val log = FileLog.getInstance(this)

        findViewById<TextView>(android.R.id.text1).text = log.log

        super.onResume()
    }
}
